apps/web/screens/Home.tsx

```diff
@@ -1,12 +1,16 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import React from 'react'
+import React, { useContext } from 'react'
 import { Box, Stack, Inline, Tag } from '@island.is/island-ui/core'
-import { Categories, SearchInput, LatestNewsSection } from '../components'
-import { useI18n } from '../i18n'
-import { Screen } from '../types'
-import { useNamespace } from '../hooks'
-import routeNames from '../i18n/routeNames'
-import FrontpageTabs from '../components/FrontpageTabs/FrontpageTabs'
+import {
+  Categories,
+  SearchInput,
+  LatestNewsSection,
+} from '@island.is/web/components'
+import { useI18n } from '@island.is/web/i18n'
+import { Screen } from '@island.is/web/types'
+import { useNamespace } from '@island.is/web/hooks'
+import routeNames from '@island.is/web/i18n/routeNames'
+import FrontpageTabs from '@island.is/web/components/FrontpageTabs/FrontpageTabs'
```

Comment:
is this imported from '@island.is/web/components' ? if so maybe add this to the '@island.is/web/components' import above